prow/spyglass/gcsartifact_fetcher_test.go

```diff
@@ -94,7 +95,7 @@ func TestArtifacts_ListGCS(t *testing.T) {
 	}{
 		{
 			name:   "Test ArtifactFetcher simple list artifacts",
-			source: "test-bucket/logs/example-ci-run/403",
+			source: "gs://test-bucket/logs/example-ci-run/403",
```

Comment:
Same, please leave the existing tests and add new ones

Reply:
Same as above. Was only used with scheme, but can handle both now (also extended the tests)